src/app/pages/jails/jail-add/jail-add.component.ts

```diff
@@ -59,53 +60,61 @@ export class JailAddComponent implements OnInit {
       type: 'checkbox',
       name: 'dhcp',
       placeholder: T('DHCP autoconfigure IPv4'),
-      tooltip: T('Check to start the jail with the Dynamic Host\
- Configuration Protocol enabled. <b>VirtIO</b> and <b>Berkeley Packet\
- Filter</b> must also be enabled.'),
+      tooltip: T('Set to start the jail with the Dynamic Host \
+                 Configuration Protocol enabled. <b>VirtIO</b> and \
+                 <b>Berkeley Packet Filter</b> must also be enabled.'),
     },
     {
       type: 'checkbox',
       name: 'vnet',
       placeholder: T('VirtIO Virtual Networking'),
-      tooltip: T('Check to use VirtIO to emulate network devices for\
- this jail.  Check this if a fully virtualized per-jail network stack is\
- required. See <a\
- href="https://www.freebsd.org/cgi/man.cgi?query=virtio&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
- target="_blank">VIRTIO(4)</a> for more details.'),
+      tooltip: T('Set to use <a \
+href="https://www.freebsd.org/cgi/man.cgi?query=virtio&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
```

Comment:
Adjust spacing here to line text up with the first character of the tooltip:
```
tooltip: T('Set to use <a \
                href="https://www.freebsd.org/cgi/man.cgi?query=virtio&manpath=FreeBSD+11.1-RELEASE+and+Ports"\
                target="_blank">VirtIO(4)</a> to emulate network devices for the jail.
```

Reply:
if i line up URLs to be in line with the rest of the tooltip, the URL broke. At least they did in my testing, that's why all the URLs are over to the far left. When I moved the url to line up with where the tooltip should be, an extra space is introduced and broke the link,

Reply:
I've fixed the formatting on all of the URLs to line up with the tooltips properly now.